Test/Fragments/ContactFragment.cs

```diff
@@ -0,0 +1,175 @@
+            }.Select(x => view.FindViewById<CheckBox>(x)).ToList();
+
+            checkBoxs.ForEach(x => x.Click += CheckBox_Click);
+
+            view.FindViewById<Button>(Resource.Id.buttonSend).Click += ButtonSend_Click;
+
+            firstName = view.FindViewById<EditText>(Resource.Id.textViewFirstName);
+            lastName = view.FindViewById<EditText>(Resource.Id.textViewLastName);
+            email = view.FindViewById<EditText>(Resource.Id.textViewEmail);
+            company = view.FindViewById<EditText>(Resource.Id.textViewCompany);
+            position = view.FindViewById<EditText>(Resource.Id.textViewPosition);
+            branch = view.FindViewById<EditText>(Resource.Id.textViewBranch);
+            comments = view.FindViewById<EditText>(Resource.Id.textViewComments);
+
+            spinner = view.FindViewById<Spinner>(Resource.Id.spinner1);
+            spinner.Adapter = new ArrayAdapter<string>(view.Context, Android.Resource.Layout.SimpleSpinnerItem, new string[] { "Frédéric", "Markus", "Paul", "Kay", "Joël", "Pascal", "Mike", "Hoa Vo" });
+
+            return view;
+        }
+
+        private void ButtonSend_Click(object sender, EventArgs e)
+        {
+            if (Check())
+            {
+                contact.Reason = reason;
+                contact.HelloTo = checkBoxs[2].Checked ? spinner.SelectedItem.ToString() : "";
+                contact.FirstName = firstName.Text;
+                contact.LastName = lastName.Text;
+                contact.Email = email.Text;
+                contact.Company = company.Text;
+                contact.Position = position.Text;
+                contact.Industry = branch.Text;
+                contact.Comments = comments.Text;
+                contact.Token = "v5?1Q$Q974-tE_i1K!_!7qoiEo_?4@-35ptR57tedi66Mx-Duqm-2?x$j@zX6@U";
+                contact.Images = string.IsNullOrEmpty(imageBase64) ? new List<Image>() : new List<Image>
+                {
+                    new Image
+                    {
+                        Id = 0,
+                        ImageBase64 = imageBase64
+                    }
+                };
+
+                try
+                {
+                    Swagger.Instance.PostContact(contact);
+                    Toast.MakeText(view.Context, "Sent", ToastLength.Short).Show();
+                }
+                catch (WebException webException)
+                {
+                    Log.Debug("ApiError", $"Exception: {webException.Message}");
+                    Toast.MakeText(view.Context, "Send failed", ToastLength.Short).Show();
+                }
+            } else
+            {
+                Toast.MakeText(view.Context, "Value empty", ToastLength.Short).Show();
+            }
+        }
+
+        private bool Check() => checkBoxs.Any(x => x.Checked) && !string.IsNullOrEmpty(firstName.Text) && !string.IsNullOrEmpty(lastName.Text) && !string.IsNullOrEmpty(email.Text);
+
+        private void InitCheckBoxs()

```

Comment:
The method name can be better `UncheckAllCheckBoxs`

Reply:
Done